lib/web_ui/test/text/font_collection_test.dart

```diff
@@ -0,0 +1,108 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
thanks for adding a test for font loading!

Reply:
thanks!